### ISSUE TITLE
how to get ender pearl and nether star

### ISSUE THREAD

**Issue by HoleFish** [hjae78/Gregicality-Community-Pack#78]
These mob loot cant be obtained for hostile mobs are disabled.The only way to get ender pearl is to trade with cleric,which is expensive and unsustainable.Absolutely no way to get nether star.

**Comment by htmlcsjs** [comment]
Have you checked jei for the ender pearl recipe. Also nether star should be obtainable via mining when the GC update is out

**Comment by Syrcan** [comment]
Thank you for leaving feedback.
Mob drops alternatives have been partially introduced.

The current way to obtain ender pearls is: make enderpearl dust in chemical reactor, then put the dust in implosion compressor. We will be sure to add quests once wenre certain we want to stick with this.

Nether Stars become mandatory for progression starting in IV age, if I remember correctly. We have plans to make it available from ores in a space dimension.

**Comment by HoleFish** [comment]
Isnt gcm1.2 the latest? I cant find any recipe to make enderpearl dust in chemical reactor;instead it's in mixer,and it needs Be from End

**Comment by Syrcan** [comment]
Right, that's a mixer recipe. With the newer ore generation, beryllium will spawn in the Nether.
